agents/postgres/pgstatmonitor/pgstatmonitor.go

```diff
@@ -356,6 +361,45 @@ func (m *PGStatMonitorQAN) getNewBuckets(ctx context.Context, periodLengthSecs u
 	return buckets, nil
 }
 
+func (m *PGStatMonitorQAN) checkErrorsView(ctx context.Context) error {
+	row := &pgStatMonitorErrors{}
+	rows, err := m.q.SelectRows(pgStatMonitorErrorsView, "")
+	if err != nil {
+		return errors.Wrap(err, "failed to query pg_stat_monitor_errors view")
+	}
+
+	now := time.Now()
+	for ctx.Err() == nil {
+		if err = m.q.NextRow(row, rows); err != nil {
+			if errors.Is(err, reform.ErrNoRows) {
+				break
+			}
+
+			return errors.Wrap(err, "cannot read row from errors view")
+		}
+
+		messageTime, err := time.Parse("2006-01-02 15:04:05", row.MessageTime)
+		if err != nil {
+			return errors.Wrap(err, "cannot parse messageTime")
+		}
+		if now.After(messageTime) {
+			continue
+		}
+
+		template := "Message: %s, Calls: %d"
+		switch row.Severity {
+		case "INFO":
+			m.l.Infof(template, row.Message, row.Calls)
+		case "WARNING":
+			m.l.Warningf(template, row.Message, row.Calls)
+		case "ERROR":
+			m.l.Errorf(template, row.Message, row.Calls)
```

Comment:
Default to catch unknown severity?

Reply:
There is no possibility to have unkown severity expect error in PGSM side:
https://github.com/percona/pg_stat_monitor/pull/107/files#diff-0b65d2614dea695d2f1f5d15e1ad191af65edd37f0e8d6be4de030cb36d88aa1R266

I can add default one if its requested.